src/OpenTelemetry.Instrumentation.AWS/Implementation/AWSTracingPipelineHandler.cs

```diff
@@ -136,25 +139,27 @@ private static void ProcessException(Activity activity, Exception ex)
 #endif
     private static void AddRequestSpecificInformation(Activity activity, IRequestContext requestContext, string service)
     {
-        if (AWSServiceHelper.ServiceParameterMap.TryGetValue(service, out var parameter))
+        AmazonWebServiceRequest request = requestContext.OriginalRequest;
+        if (AWSServiceHelper.ServiceParameterMap.TryGetValue(service, out var parameters))
```

Comment:
Can you please move bedrockAgent logic (the code below) inside here? For example, if it is bedrockAgent service, we extract no params directly, instead, we check the operation name and extract from it's context as what you implemented below.